ui-v2/app/styles/components/composite-row/layout.scss

```diff
@@ -29,7 +29,7 @@
   align-self: center;
 }
 %composite-row-icon {
-  margin-right: 10px;
+  margin-right: 6px;
```

Comment:
Discussed with Jasmine to have the spacing between health-check icon and text be 6px, atm. In some frames it's 4px and in other it's 10px.